src/script/components/MessagesList/Message/ContentMessage/MessageActions/MessageReactions/MessageReactionsList.tsx

```diff
@@ -17,113 +17,43 @@
   return (
-    <div css={messageReactionWrapper}>
+    <div css={messageReactionWrapper} data-uie-name="message-reactions">
       {reactionsList.map(([emoji, users], index) => {
         const emojiUnicode = getEmojiUnicode(emoji);
-        const emojiUrl = getEmojiUrl(emojiUnicode);
-        const emojiName = getEmojiTitleFromEmojiUnicode(emojiUnicode);
-        const emojiCount = users.length;
-        const isActive = isSelectedEmoji === emojiUrl;
+        const emojiListCount = reactionsList.length;
+
         return (
-          <Fragment key={emojiUnicode}>
-            <Tooltip
-              body={
-                <div css={messageReactionButtonTooltip}>
-                  <EmojiImg
-                    emojiImgSize={{
-                      width: '1.2rem',
-                    }}
-                    styles={messageReactionButtonTooltipImage}
-                    emojiUrl={emojiUrl}
-                    emojiName={emojiName}
-                  />
-                  <p css={messageReactionButtonTooltipText}>
-                    <span
-                      onClick={onTooltipReactionCountClick}
-                      onKeyDown={event => {
-                        if (!isEnterKey(event)) {
-                          onTooltipReactionCountClick();
-                        }
-                      }}
-                      role="button"
-                      tabIndex={0}
-                      css={messageReactionButtonTooltipTextLink}
-                    >
-                      {t('conversationLikesCaption', {number: emojiCount.toString()})}
-                    </span>{' '}
-                    {t('conversationLikesCaptionReacted', {emojiName})}
-                  </p>
-                </div>
-              }
-            >
-              <button
-                css={{...messageReactionButton, ...getReactionsButtonCSS(isActive)}}
-                aria-label={t('messageReactionDetails', {emojiCount: emojiCount.toString(), emojiName})}
-                title={emojiName}
-                aria-pressed={isActive}
-                type="button"
-                tabIndex={messageFocusedTabIndex}
-                className="button-reset-default"
-                onClick={() => {
-                  setSelected(emojiUrl);
-                  handleReactionClick(emoji);
-                }}
-                onKeyDown={event => {
-                  // is last reaction then on tab key press it should hide the reaction menu
-                  if (index === reactionsList.length - 1) {
-                    if (!event.shiftKey && isTabKey(event)) {
-                      onLastReactionKeyEvent();
-                    }
-                  }
-                }}
-              >
-                <EmojiImg emojiUrl={emojiUrl} emojiName={emojiName} />
-                <span css={messageReactionCount(isActive)}>{emojiCount}</span>
-              </button>
-            </Tooltip>
-          </Fragment>
+          <div key={emojiUnicode}>
```

Comment:
Do we need this div? Can't we just use key prop on component itself? <EmojiPill key={...} />